src/main.c

```diff
@@ -189,8 +189,6 @@ static bool handle_rx_data(const uint8_t *buf_in, uint16_t buf_in_size)
 
 void tud_vendor_rx_cb(uint8_t itf, const uint8_t *buf_in, uint16_t buf_in_size)
 {
-	(void)itf;
```

Comment:
commit `Ensure dln2 IF is ignoring DAP IF traffic`: how does removing this line achieve that?